common/lib/protocol-definitions/src/summary.ts

```diff
@@ -3,8 +3,14 @@
  * Licensed under the MIT License.
  */
 
+/**
+ * TODO
```

Comment:
```suggestion
 * object representing a node within a summary tree. If a node is a {ISummaryTree} it can contain additional SummaryObject(s) as its children.
```